codespell_lib/data/dictionary.txt

```diff
@@ -31759,7 +31822,9 @@ linez->lines
 lingth->length
 linheight->lineheight
 Linix->Linux
-linke->linked, like,
+linke->linked, link, like, line,
```

Comment:
Add `links` to the list? ad `minke` as in "minke whale"?

Reply:
I'll add `links`. I couldn't find `minke`.